runit-elixir-focal/1.13.4/collectd-socket.conf

```diff
@@ -0,0 +1,8 @@
+LoadPlugin unixsock
```

Comment:
Why didn't we need this before?

Reply:
That file did exist before, there just wasn't a 1.13.4 subdirectory